Makefile

```diff
@@ -68,12 +75,24 @@ test/e2e: $(OPERATOR_SDK)
 		--no-setup \
 		--go-test-flags "$(E2E_TEST_ARGS)"
 
-
 $(local_handler_manifest): deploy/operator.yaml
 	mkdir -p $(dir $@)
 	sed "s#REPLACE_IMAGE#$(LOCAL_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)#" \
 		deploy/operator.yaml > $@
 
+
+$(versioned_operator_manifest): HANDLER_IMAGE_SUFFIX = :$(shell hack/version.sh)
+$(versioned_operator_manifest): version/version.go
+	mkdir -p $(dir $@)
+	sed "s#REPLACE_IMAGE#$(HANDLER_IMAGE)#" \
+		deploy/operator.yaml > $@
+
+$(): deploy/operator.yaml
```

Comment:
what is this?

Reply:
Brain fart, it was the local version of the operator.yaml, removed by accident.